exampleSite/config.toml

```diff
@@ -50,63 +50,37 @@ googleAnalytics = ""
 # Keep an eye out for updates. These will be moved out to make navbar more
 # configurable.
 [[menu.main]]
-  url = "/blog"
-  name = "Blog"
+  url = "/"
+  name = "Home"
   weight = 10
 
 [[menu.main]]
   url = "/fragments"
   name = "Fragments"
   weight = 20
 
-[[menu.main]]
-  url = "/about-split"
-  name = "Page-split"
-  weight = 30
-
 [[menu.main]]
   url = "/about"
-  name = "Page-single"
-  weight = 40
-
-[[menu.main]]
-  url = "#contact"
-  name = "Contact"
-  weight = 50
+  name = "About"
+  weight = 30
 
 [[menu.footer]]
-  url = "#"
-  name = "Link"
+  url = "https://github.com/okkur/syna/blob/master/docs/README.md"
+  name = "Getting Started"
   weight = 10
 
 [[menu.footer]]
-  url = "#"
-  name = "Long Link"
-  weight = 30
-
-[[menu.footer]]
-  url = "#"
-  name = "Tiny Link"
-  weight = 30
+  url = "https://about.okkur.io/"
+  name = "About Okkur Labs"
+  weight = 20
 
 [[menu.footer_social]]
   weight = 30
-  name = "linkedin"
-  pre = "fab fa-linkedin-in"
+  name = "github"
+  pre = "fab fa-github"
   url = "#"
```

Comment:
Needs the correct url, but didn't even think of that solution to just use the social icons feature. Awesome.